tool/vfx/vfxSection.h

```diff
@@ -1190,6 +1190,10 @@ class SectionShaderOption : public Section
 #if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 35
         INIT_STATE_MEMBER_NAME_TO_ADDR(SectionShaderOption, disableLicm, MemberTypeBool, false);
 #endif
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 39
+        INIT_STATE_MEMBER_NAME_TO_ADDR(SectionShaderOption, unrollThreshold, MemberTypeInt, false);
+#endif
+
```

Comment:
I think you need to adjust the value of MemberCount
MemberCount = 15 -> MemberCount = 16